foreman/data_refinery_foreman/foreman/management/commands/run_tximport.py

```diff
@@ -55,9 +53,9 @@ def run_tximport():
                 processor_job.pipeline_applied = tximport_pipeline.value
                 processor_job.ram_amount = 8192
                 # This job doesn't need to run on a specific volume
-                # but it uses the same Nomad job as Salmon jobs which
+                # but it uses the same Batch job as Salmon jobs which
                 # do require the volume index.
-                processor_job.volume_index = random.choice(list(get_active_volumes()))
+                processor_job.volume_index = choose_job_queue
```

Comment:
does this need to be execute?

Reply:
That's a good catch. I've already made the change in my latest branch that's based off this one. I'll just add it in that one so I don't have to deal with conflicts trying to add it to this one.